components/Player.js

```diff
@@ -20,8 +20,12 @@ export default function Player(props) {
       <div className=" flex w-full justify-between">
         <div className="flex ">
           <div className={`flex items-center justify-center rounded-lg p-2`}>
+            if(props.bgColor != undefined) {}
             <div
-              className={`bg-slate-300 ${props.bgColor} flex justify-center rounded-full h-12 w-12`}
+              className={`bg-slate-300 flex justify-center rounded-full h-12 w-12`}
+              style={{
```

Comment:
I think it is better to keep styling consistent using the tailwind class for background color. In this case, if you want to check if the color != undefined you can simply make the check in the className literal like this: 

${props.bgColor? props.bgColor : 'bg-slate-300'}

Reply:
Thanks for the feedback! I got confused since the users already created in the server had color codes for the background instead of Tailwind classes.

Reply:
I made an update so it uses both, tailwind and color codes

Reply:
What is the expected behaviour here? Should every subsequent user added have different colors? As it stands, every subsequenet user has the background color set to slate-500. 
![image](https://user-images.githubusercontent.com/109530511/181345917-19121624-bb65-429f-83cb-f326433a977a.png)

Reply:
Only Ben and Tommy should have custom colors

Reply:
The others should grey and the active user should be pink

Reply:
There isn't an option yet for choosing your own color